### ISSUE TITLE
Possible ProvONE Typos

### ISSUE THREAD

**Issue by ThomasThelen** [DataONEorg/sem-prov-ontologies#50]
The [provone:Port](https://github.com/DataONEorg/sem-prov-ontologies/blob/master/provenance/ProvONE/v1/owl/provone.owl#L319), [provone: controls](https://github.com/DataONEorg/sem-prov-ontologies/blob/master/provenance/ProvONE/v1/owl/provone.owl#L46), and [provone:Controller](https://github.com/DataONEorg/sem-prov-ontologies/blob/master/provenance/ProvONE/v1/owl/provone.owl#L365) have two entries of `prov:category`. The extra `prov:category` might supposed to be an `rdfs:comment`.

**Comment by amoeba** [comment]
Thanks @ThomasThelen. I had a look at this and submitted a pr in #58 to address these as I agree with you. Waiting on others to comment.

**Comment by mbjones** [comment]
@csjx @PaoloMissier @ludaesch Do you agree with this change? It would be the first change needed since provone_v1_dev was tagged. We have never released a v 1.0 of ProvONE, and so this might be a good time to find any cleanup needed and publish 1.0 with documentation and errata. Thoughts?

**Comment by csjx** [comment]
I looked at it, and yeah, totally agree. To do a release, I feel like we need to look through the documentation, particularly the examples, and make sure we've covered all of the pertinent concepts sufficiently.

**Comment by amoeba** [comment]
Merged #58 for now and letting this issue auto-close.

Sounds like we need some cycles in order to do a release. I'd vote for waiting for now and coming back to the idea whenever we do PROV work in the future and dovetailing in a release.

**Comment by ludaesch** [comment]
Hi team,
Sorry I'm seeing this only now. I'm having a hard time understanding all
the details and repercussions without a bit more context.

For those ProvONE enthusiasts among us - would it be more efficient to set
up a brief chat / zoom call in the near future?

**Comment by mbjones** [comment]
@ludaesch One of the reasons I think it would be good to clean it up is that yesterday we just released version 1.2 of the Science on schema.org Dataset guidelines, and those explicitly incorporate PROV and ProvONE as the recommended way to document provenance relationships in schema.org. See https://github.com/ESIPFed/science-on-schema.org/blob/master/guides/Dataset.md#provenance-relationships  So there's a good chance we will start seeing greater attention to the ProvONE vocabulary as people try to follow those guidelines. A version of the vocabulary is browseable on the Community Ontology Repository (COR) for ESIP: http://cor.esipfed.org/ont?iri=http://purl.dataone.org/provone/2015/01/15/ontology%23 

So I guess my question is whether there are any further issues that need cleanup besides the documentation stuff that Bryce fixed in this issue?

**Comment by ThomasThelen** [comment]
After playing around with ProvONE in Whole Tale, some DataONE things, and the SDTL work I have a couple of opinion-based nice-to-haves.

1. Examples demonstrating SPARQL queries to make it a little more useful/easy to use out of the box. Whether it's a docker image with blazegraph, a bunch of folders - each with files and an associated provone document, and then a set of associated queries: What were the ids of all of the files that were used? What were the ids of the files produced?, etc...
2. SHACL rules that would let users validate their ProvONE documents. Without it, it's a burden to visually scan the turtle or write queries that "should work" if the ProvONE is well formed.
3. [Opinion] The examples on the documentation page are hard to follow. 
   a. Some of them reference two or three other examples, which leaves a burden of mentally visualizing the full RDF to the user. Take for example the section for `hasDefaultParam`. It references 5 different examples and I can't mentally reconstruct the rdf _and_ figure out what hasDefaultParam means at the same time (easily).
   b. There's also inconsistency in how the RDf is numbered. (minor) For example the Association class starts on line 1, while the wasInformedBy object property starts on line two. If possible, I'd vote for removing the numbering. The PROV-O page doesn't number theirs and it makes it easier to copy+paste.
   c. There's plenty of documentation on what things _are_, but not necessarily on how to use them. For example, if I want to say that the value "2" was passed from one port to another-does it go in the channel connecting them? Does it go in each port? Does it actually belong in an Entity which is linked to the port?
   d. They should be expanded (related to c). For example, the Workflow object. The notes on the example are `The Workflow is given the string wf1 as an identifier in line 4. In addition, it is given the string ModelComparison as a descriptive title in line 5.`. This should be fairly clear for anyone reading it through the turtle and doesn't tell me anything particularly useful about a provone:Workflow, which I'm trying to learn about.
   e. The `http://example.com/` IRI using the ` :` namespace (See section `1.4 Namespaces`). I'm not sure if this is used anywhere but I think it would be confusing to see a namespace as `:` which is a fairly special character.
